apps/next-docs/content/components/SubdomainNavBar/index.mdx

```diff
@@ -113,19 +113,26 @@ const App = () => {
 render(App)
 ```
 
+## Accessibility
+
+When using the `SubdomainNavBar`, use the `onMobileMenuToggle` prop to detect when the mobile menu is opened or closed.
+
+When the menu is open, ensure that the rest of the document is hidden from screen readers. This can be achieved by adding `aria-hidden="true"` or `inert` to the main content area when the menu is open.
```

Comment:
Let's switch the order of these, so that we set context before offering the solution. Can we also move the `onMobileMenuToggle` part to the react page, as an example. The interface guidelines tab is typically implementation-agnostic.

```suggestion
When the menu is open, ensure that the rest of the document is hidden from screen readers. This can be achieved by adding `aria-hidden="true"` or `inert` to the main content area when the menu is open.
```

Reply:
We don't currently have separate Interface Guidelines and React pages for the SubdomainNavBar. The page that we have at the moment is essentially the React page as it has examples, prop tables, etc, and no real guidelines.